### DISCUSSION TITLE
Top Five Questions The Research is Answering

### DISCUSSION THREAD

**Discussion by TamaraGR** [TamaraGR/Police_Violence_Analysis#5] [Ideas]
Team, let's agree on the top five questions here

**Comment by TamaraGR** [comment]
1. What is police reaction in percentages to civilian of different races  depending on various factors (mental illness, fleeing, armed.. what else? ) (i.e. if a white person is armed, do they get killed likely?) 
2. Is there a correlation between bodycam and length of training and the murder rate?

**Comment by acfthomson** [comment]
1.  What activities was the victim doing that led up to their death?

2.  Is there a correlation between a law enforcement agency's training budget vs. shooting incidents?

3. What is the rate of positive identification of the victim having a weapon in their hand?

4. Do more diverse law enforcement agencies have lower rates of shooting? - Not sure if our data set will answer this

5. What are the rates of police violence in other countries where law enforcement carries weapons? - Not sure if our data set will answer this

**Comment by nigelrowser** [comment]
Basic Questions

1. How many police killings by race (break down by state, city, county)
2. Total amount of police killings (break down by year)

1. Correlation between age of victims and if they tried to flee

**Comment by TamaraGR** [comment]
Ok, team, good questions, let's narrow down to five and let's decide during the breakout rooms

**Comment by TamaraGR** [comment]
1. Doughnut chart of overall records for deaths by race

**Reply by TamaraGR** [reply]
use the current dataset

**Reply by TamaraGR** [reply]
2. Whatever we are figuring out with ML (one of however many visuals)

**Reply by TamaraGR** [reply]
3. A heatmap (geographical map) - with the current dataset (How many police killings by race (break down by state, city, county))

**Reply by TamaraGR** [reply]
4. Correlation between age of victims and if they tried to flee. - with the main database

**Reply by TamaraGR** [reply]
5. Correlation between budgets and killing rates (file to add)

**Reply by TamaraGR** [reply]
the file is in slack (and also on the database repo)

**Reply by shivam0921** [reply]
> 1. Doughnut chart of overall records for deaths by race

This is done
